web/src/features/moreCast2/util.ts

```diff
@@ -136,13 +143,24 @@ export const fillGrassCuring = (rows: MoreCast2Row[]): MoreCast2Row[] => {
   }
 
   for (const row of rows) {
-    if (validForecastPredicate(row)) {
-      const stationInfo = stationGrassMap.get(row.stationCode)
+    const stationInfo = stationGrassMap.get(row.stationCode)
 
-      if (stationInfo) {
-        row.grassCuring = stationInfo.grassCuring
-      }
+    if (stationInfo && row.grassCuringForecast) {
+      row.grassCuringForecast.value = stationInfo.grassCuring
     }
   }
   return rows
 }
+
+export const fillGrassCuringForward = (editedRow: MoreCast2Row, allRows: MoreCast2Row[]) => {
```

Comment:
Would you mind adding comments explaining the intended purpose for `fillGrassCuring` and `fillGrasCuringForward`?